playbooks/switch_role.yaml

```diff
@@ -1,3 +1,7 @@
+- import_playbook: sync-config-files.yaml
+  when: labsync_enabled|default(false)|bool
+- import_playbook: check-nodepool-periodic.yaml
+  when: labcheck_enabled|default(false)|bool
```

Comment:
when switch role, the cron task in the old master node should be cleaned. Has your playbook contained this part?

Reply:
Yes, the playbook will setup the task in master nodes and delete the task in slave nodes

Reply:
ah, I find it.

Reply:
@wangxiyuan Yes
The labsync will tear down the crontab in nodepool-launcher-slave:zuul-scheduler-slave [1], and the labcheck will tear down the crontab in nodepool-launcher-slave. [2]

[1] https://github.com/theopenlab/labkeeper/blob/master/playbooks/sync-config-files.yaml#L34-L44
[2] https://github.com/theopenlab/labkeeper/blob/master/playbooks/check-nodepool-periodic.yaml#L13-L20